src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Badge/Badge.ts

```diff
@@ -118,11 +121,23 @@ export var bindPath = (
             "data", SIBadgeAssignment.nick, "badge"
         ]);
 
-        $q.all(_.map(assignableBadgePaths, (b) => adhHttp.get(b).then(extractBadge))).then((badges) => {
-            scope.badges = badges;
-            var groupPaths : string  = _.union.apply(_, _.map(badges, "groups"));
+        $q.all(_.map(assignableBadgePaths, (b) => adhHttp.get(b).then(extractBadge))).then((badges : any) => {
+            scope.badges = _.keyBy(badges, "path");
+            var groupPaths : string[]  = _.union.apply(_, _.map(badges, "groups"));
```

Comment:
Related: https://stackoverflow.com/questions/1986896/what-is-the-difference-between-call-and-apply

Summary: `_.union.apply(_, [[1, 2], [1]]) == _.union([1, 2], [1]) != _.union([[1, 2], [1]])`